javaparser-core/src/main/java/com/github/javaparser/ast/stmt/SwitchEntry.java

```diff
@@ -98,23 +97,24 @@ public enum Type {
     private boolean isDefault;
 
     public SwitchEntry() {
-        this(null, new NodeList<Expression>(), Type.STATEMENT_GROUP, new NodeList<>());
+        this(null, new NodeList<Expression>(), Type.STATEMENT_GROUP, new NodeList<>(), false);
     }
 
     @AllFieldsConstructor
-    public SwitchEntry(final NodeList<Expression> labels, final Type type, final NodeList<Statement> statements) {
-        this(null, labels, type, statements);
+    public SwitchEntry(final NodeList<Expression> labels, final Type type, final NodeList<Statement> statements, final boolean isDefault) {
```

Comment:
I think this is generated by the generators. This is a rather obscure part of JP for me as I've never used it and it's not well documented. As far as I know, the link to the documentation, which can be accessed via the wiki, is https://javaparser.org/code-generation-and-maven-in-javaparser/

Reply:
I added the `isDefault` field and then ran the generators as per the instructions in that page, but this didn't update the 3 constructors in this file. Everything else was correctly generated however.

Reply:
> but this didn't update the 3 constructors in this file

I think this is a logical behaviour as they are certainly not annotated and they have probably been created by providing shortcuts with default values.